eclipse/plugins/org.switchyard.tools.ui/src/org/switchyard/tools/ui/operations/CreateSwitchYardProjectOperation.java

```diff
@@ -795,6 +795,13 @@ private Model createPom() {
                         dependency.setVersion(versionString);
                     } // else don't add it
                 }
+                if (isIntegration) { // hack for SWITCHYARD-2936
+                    if (dependency.getArtifactId().equalsIgnoreCase("switchyard-component-bpm") //$NON-NLS-1$
```

Comment:
same commen tthan for SwitchYardProjectWorkingCopy, maybe we can factorize these 3 lines of code